src/styles.css

```diff
@@ -264,6 +265,26 @@ h3 {
   height: 97%;
 }
 
+.individual-recipe-image {
+  position: absolute;
+  bottom:0;
+  right:0;
+  flex-shrink: 0;
+  width: 200px;
+  height: 200px;
+  overflow: hidden;
+  border-radius: 50%;
+  z-index: 1;
```

Comment:
my only curiosity design wise is to check what happens if z-index is same for the title and the image? Does that look better? I leave the creative decision to you but would like to know why you chose one over the other.

Reply:
Looks like if they are the same, the image will be on top of the title.  Maybe because it comes second in the html?